meshtastic/mesh.proto

```diff
@@ -525,6 +525,12 @@ enum HardwareModel {
    * Adafruit NRF52840 feather express with SX1262, SSD1306 OLED and NEO6M GPS
    */
   TWC_MESH_V4 = 62;
+
+  /*
+   * PROMICRO_DIY_M(M)
+   * Promicro NRF52840 with SX1262/LLCC68, SSD1306 OLED and NEO6M GPS
+   */
+  PROMICRO_DIY = 63;
```

Comment:
Let's call it NRF52_PROMICRO_DIY. There are more pro micro form factor boards that are non NRF, so I want to be clear in the hw model name.

Reply:
Do I need rename variants too?